Python/frozen_modules/hello.h

```diff
@@ -1,4 +1,4 @@
-/* Auto-generated by Programs/_freeze_importlib.c */
+/* Auto-generated by Programs/_freeze_module.c */
```

Comment:
I think we have to do something to make GitHub recognize all frozen .h files as generated. I think there's something in .github/workflows.yml.

Reply:
The only applicable thing under .github/ is the "check_generated_files" job in .github/workflows/build.yml.  However, nothing explicit has to be done there since it only runs "make regen-all" and makes sure nothing changes.

However, .gitattributes does need to be updated.  Thanks for pointing this out!